composer.json

```diff
@@ -1,5 +1,8 @@
 {
     "name": "namelesscoder/uncache",
     "description": "TYPO3 Cache Crippler",
-    "type": "typo3-cms-extension"
+    "type": "typo3-cms-extension",
+    "require": {
+        "typo3/cms-core": "^7.6 || ^8.7 || ^9.4"
```

Comment:
Should probably be `^9.5` for the last one ;)

If you want to follow up with a FEATURE that adds `^10` here and in ext_emconf.php as well that would be very appreciated! I'm pretty sure the ext is compatible with v10 even though the constraints don't yet mention it.

Reply:
Switched to ^9.5. v10 will try to test this break haha.